### PR TITLE
Simplify README file

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -75,7 +52,7 @@ To add a new feature to PeleC, the procedure is:
     git checkout development
     git pull                     [fix any identified conflicts between local and remote branches of "development"]
     git checkout AmazingNewFeature
-    git merge development        [fix any identified conflicts between "development" and "AmazingNewFeature"]
+    git rebase development        [fix any identified conflicts between "development" and "AmazingNewFeature"]
```
Comment (marchdf) [inline README.rst:55]: yay for this!

File: README.rst [hunk 2]
```diff
@@ -57,7 +34,7 @@ Alternatively, one can set environment variables to use AMReX and PelePhysics re
 Dependencies
 ~~~~~~~~~~~~
 
-`PeleC` was created as a renamed, stripped down version of `Maui`, and is built on the `AMReX` library.  In the process, the Microphysics folder was extracted, and reorganized into a separate repository, `PelePhysics`.  
+`PeleC` was created as a renamed, stripped down version of `Maui`, and is built on the `AMReX` library.  In the process, the Microphysics folder was extracted, and reorganized into a separate repository, `PelePhysics`. PeleC requires the `SUNDIALS <https://github.com/LLNL/sundials>`_ ODE solver library.
```
Comment (marchdf) [inline README.rst]: Should we remove this Maui stuff? Doesn't really feel relevant anymore... Just say that we depend on PelePhysics for transport, EOS, and chem?

Comment (jrood-nrel) [reply README.rst]: Agreed.

File: README.rst [hunk 3]
```diff
@@ -14,34 +14,11 @@ Getting Started
...
-
-    export PELEC_HOME=<location for PeleC>
-    git clone git@github.com:AMReX-Combustion/PeleC.git ${PELEC_HOME}
-
-4. Move to an example build folder, build an executable, run a test case: ::
-
-    cd ${PELEC_HOME}/Exec/RegTests/PMF
-    make
+    make TPL && make -j
     ./Pele3d.xxx.yyy.ex inputs_ex
```
Comment (marchdf) [inline README.rst]: We renamed all the `input_ex` stuff right? Let's update here.

**Review by SreejithNREL — APPROVED** [review]
Perfect! Builds now. Thanks

**Review by marchdf — APPROVED** [review]

**Pull request merged**